src/Php55/composer.json

```
@@ -16,17 +16,15 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "ircmaxell/password-compat": "~1.0"
+        "php": ">=7.1"
```

On the metapackage we could even go lower:

```suggestion
        "php": ">=5.6"
```

But the current constraint is fine as well.

---

Nope: that would conflict with what symfony/polyfill tells.

---

All right.